src/wireviz/DataClasses.py

```diff
@@ -271,6 +271,7 @@ class Cable:
     show_name: Optional[bool] = None
     show_wirecount: bool = True
     show_wirenumbers: Optional[bool] = None
+    show_colorname: bool = True
```

Comment:
I suggest a different name:
```suggestion
    show_wirecolor_text: bool = True
```
Alternatively `show_color_text`, but then someone might expect it to also affect showing the `cable.color` (jacket color) text. Do you want that?

I suggest `text` instead of `name` because the text might be different text representations of the colors, depending on `options.color_mode`.

Reply:
This increases the code complexity slightly, but it seems easier to read, and hopefully is easier to understand as well.

I assume you'll also allow the values `True` for showing everything and `False` for showing nothing.

With `None` as default value for `show_colors`, you can set it in `__post_init__()` default `plain` for your single-core cable use case and `True` otherwise.